custom/icds_reports/ucr/data_sources/person_cases_v2.json

```diff
@@ -2395,6 +2395,47 @@
             "constant": null
           }
         }
+      },
+      {
+        "column_id": "household_id",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
```

Comment:
Can you change this to `icds_parent_id` ?

Reply:
fixed, and cleaned up the other uses of the same expression in the ucr